state/presence/pingbatcher.go

```diff
@@ -293,50 +293,40 @@ func (pb *PingBatcher) flush() error {
 	session := pb.pings.Database.Session.Copy()
 	defer session.Close()
 	pings := pb.pings.With(session)
-	bulk := pings.Bulk()
 	docCount := 0
 	fieldCount := 0
 	t := time.Now()
-	bulkCount := 0
+	slots := make(map[int64]bool)
```

Comment:
this doesn't appear to be used anywhere apart from the insertion - delete?

Reply:
ah, yeah I had been reporting it, I'll delete it as I stopped tracking it.